app/scripts/controllers/app-state.js

```diff
@@ -465,6 +466,16 @@ export default class AppStateController extends EventEmitter {
     });
   }
 
+  /**
+   * Sets shouldShowAggregatedBalancePopover to false once the user toggles
+   * the setting to show native token as main balance.
+   */
+  setAggregatedBalancePopoverShown() {
```

Comment:
Are we going to keep showing the aggregated balance popover every single time? Are there plans to eventually remove it once user's get used to this feature? The best solution would probably persisting that it has been shown so it doesn't pop up ever again but still shows for new users.

Not a blocker just a suggestion and probably out of scope

Reply:
Hi Vince 👋 ; for now users will see the popover just once. Once they close it they wont see it again.
I think it makes sense to keep for couple releases then remove it totally.